.gitignore

```diff
@@ -1,2 +1,5 @@
 .env
 vendor/
+
+# Temp/backup files
+*.swp
```

Comment:
I think it would be better to include only items related to this repo in .gitignore. For editor-specific files you can set them globally for yourself, instructions [here](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).

Reply:
✅